terraform/dev/terragrunt.hcl

```diff
@@ -1,5 +1,7 @@
 terraform {
-  source = "git::https://github.com/bcgov/startup-sample-project-aws-containers-terraform-modules.git//?ref=v0.1"
```

Comment:
I made the tag v0.2 so this can be updated

Reply:
Done